activerecord/test/cases/migration_test.rb

```diff
@@ -942,8 +942,7 @@ def migrate(x)
         }.new
 
         migrator = ActiveRecord::Migrator.new(:up, [migration], @schema_migration, 100)
-
-        query = "SELECT COUNT(*) FROM pg_stat_activity WHERE datname = '#{ActiveRecord::Base.connection_db_config.database}'"
+        query = "SELECT COUNT(*) FROM pg_stat_activity WHERE datname = '#{ActiveRecord::Base.connection_db_config.database}' AND state = 'idle'"
```

Comment:
@eileencodes  I couldn't reproduce this locally even with the seeds from CI. However, this additional filter will only return the idle queries which is what this test cares about.